docs/pages/en-us/Features.md

```diff
@@ -13,29 +13,24 @@ banner:
 
 # Features
 
-The Audit service maintains a documented audit trail of user and system activity in a centralized database. The benefits of using the Audit service include traceability and accountability, but the audit records can also be used for analytical, archiving and reporting purposes. 
+The Audit service maintains a documented audit trail of user and system activity in a centralized database. The benefits of using the Audit service include traceability and accountability, but the audit records can also be used for analytical, archiving and reporting purposes.
 
 ## Support for Multiple Applications
-User and system audit events are defined per application. The Audit service offers an easy and user-friendly way to register these events using the Audit Management web service API.
+User and system audit events are defined per application.
 
 ## Multi-Tenant Aware
-The service supports multiple tenants in a secure and scalable way. These tenants are registered using the Audit Management web service API. Multi-tenancy support also facilitates audit reporting on a per tenant basis. 
+The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis. 
```

Comment:
Updated to the same as above: "The service supports multiple tenants in a secure and scalable way.  Multi-tenancy support also facilitates audit reporting on a per tenant basis and across multiple tenants"